docker-plugin/src/main/java/com/nirima/jenkins/plugins/docker/DockerTemplate.java

```diff
@@ -227,10 +232,14 @@ public DockerSlave provision(StreamTaskListener listener) throws IOException, De
 
         logger.println("Launching " + image );
 
-        int numExecutors = 1;
         Node.Mode mode = Node.Mode.NORMAL;
 
-        RetentionStrategy retentionStrategy = new OnceRetentionStrategy(idleTerminationMinutes());
+        RetentionStrategy retentionStrategy = null;
+        if (numExecutors == 1) {
+            retentionStrategy = new OnceRetentionStrategy(idleTerminationMinutes());
+        } else {
+            retentionStrategy = new CloudRetentionStrategy(idleTerminationMinutes());
+        }
```

Comment:
Is it make any sense? What will happen if OnceRetentionStrategy will be used with more executors?